nx/lib/nx/binary_backend.ex

```diff
@@ -1648,7 +1707,7 @@ defmodule Nx.BinaryBackend do
   defp top_dimension_slice?(_, _, _), do: false
 
   @impl true
-  def put_slice(out, tensor, start_indices, slice) do
+  def put_slice(out, tensor, start_indices, slice, combine_fn \\ fn _prev, new -> new end) do
```

Comment:
I plan on reverting this after the better implementation for scatter_add is reached